### ISSUE TITLE
Cannot create accounts

### ISSUE THREAD

**Issue by Jeroen196** [novskey/PalletTown#24]
I just cannot get creating accounts to work.

Whenever I use a gmail address it gives 'Check email is correct and not a gmail address and try again'.
When using a hotmail address, the log returns an exception and no account is created. 

Error log:

> [16:16:51]     [DEBUG]:[Main Thread]:  initializing..
[16:16:51]     [DEBUG]:[Main Thread]: Attempting to create user user:password. Opening browser...
[16:16:51]     [DEBUG]:[Main Thread]: Proxy type: 
[16:16:51]     [DEBUG]:[Main Thread]: No 2captcha key
[16:16:51]     [DEBUG]:Step 1: Verifying age using birthday: 1956-11-10
[16:16:51]     [DEBUG]:[Main Thread]: trying to execute workaround script
[16:16:51]     [DEBUG]:[Main Thread]: done executing workaround script, submitting dob
[16:16:51]     [DEBUG]:[Main Thread]: dob submitted
[16:16:51]     [DEBUG]:[Main Thread]: Step 2: Entering account details
[16:16:51]     [DEBUG]:[Main Thread]: https://club.pokemon.com/us/pokemon-trainer-club/parents/sign-up
[16:16:51]     [DEBUG]:[Main Thread]: https://club.pokemon.com/us/pokemon-trainer-club/parents/sign-up
[16:16:51]     [DEBUG]:[Main Thread]: You did not pass a 2captcha key. Please solve the captcha manually.
[16:16:51]     [DEBUG]:[Main Thread]: Waiting on captcha
[16:16:51]     [DEBUG]:[Main Thread]: Captcha successful. Sleeping for 1 second...
[16:16:51]     [DEBUG]:[Main Thread]: trying to submit
[16:16:51]     [DEBUG]:[Main Thread]: submitted
[16:16:51]     [DEBUG]:[Main Thread]: trying to validate response
[16:16:51]     [DEBUG]:[RESPONSE_VALIDATOR]: https://club.pokemon.com/us/pokemon-trainer-club/parents/sign-up
[16:16:51]     [DEBUG]:[RESPONSE_VALIDATOR]: Username already in use
[16:16:51]     [DEBUG]:[Main Thread]: Failed to create user: welspevchangexiting...
[16:16:51]     [DEBUG]:[Main Thread]: threw failed to create user exception, terminate
[16:16:51] [Main Thread]: threw failed to create user exception, terminate
[16:16:51] Account 1 failed
[16:16:51] done
[16:16:51] 0/1 successes

When using an email address from Zoho mail, no verification mail ever arrives..

I've done above both with and without using the Auto Verify Accounts option. Is anyone else still able to create accounts? And if so, using what kind of mail?

**Comment by acymap** [comment]
Same for me... I'm unable to create new accounts. it fails everytime with the "Username already in use" error, whichever the username is...

**Comment by majuss** [comment]
Same here! Seems like they changed something on the login page...

**Comment by parislarkins** [comment]
Will be investigating in the next week or So when I'm not sick

**Comment by acymap** [comment]
The registration form seems to reject any "plusmail" address

**Comment by beloware** [comment]
Not just +mail is the issue. own doamin also not works.

`[15:01:22]     [DEBUG]:[Worker 0]: Captcha still not solved, waiting another 10 seconds.
[15:01:22]     [DEBUG]:[Worker 0]: Solved captcha
[15:01:22]     [DEBUG]:[Worker 0]: trying to submit
[15:01:22]     [DEBUG]:[Worker 0]: submitted
[15:01:22]     [DEBUG]:[Worker 0]: trying to validate response
[15:01:22]     [DEBUG]:[RESPONSE_VALIDATOR]: https://club.pokemon.com/us/pokemon-trainer-club/parents/sign-up
[15:01:22]     [DEBUG]:[RESPONSE_VALIDATOR]: Username already in use
[15:01:22]     [DEBUG]:[Worker 0]: Failed to create user: smosssmonghoskexiting...
[15:01:22]     [DEBUG]:[Worker 0]: threw failed to create user exception, terminate
[15:01:22] [Worker 0]: threw failed to create user exception, terminate
[15:01:22] Account 2 failed
[15:01:22] Worker 0done making account 2 sleeping for 5000ms
[15:01:27] Worker 0 did 1 tasks
[15:01:27] done
[15:01:27] 0/3 successes`

**Comment by parislarkins** [comment]
Yes there is a problem with 2captcha creation in pallettown atm, however from limited testing if doesn't seem that the URL is the issue. When I get time I will be remaing PalletTown and these issues will be fixed, sorry about that

**Comment by ultrafunkamsterdam** [comment]
The + mail is working fine in the latest version using zoho with own domain.  You just need to set threads to 1 and waiting time to 30 seconds. You'll have 2 accounts per minute. Created 200 last friday.

**Comment by tuthanhhoang** [comment]
URL  2capcha error.
Important!
i received notice from 2capcha
We've found that you still don't send us the URL or the website where reCAPTCHA is being solved.
At the moment when we get the URL from the customer we forward our worker to the website where he gets the reCAPTCHA, solves it and then returns a token to us.
In cases when we don't get the URL of the website from you, we use a small exploit in reCAPTCHA to get the token. But our insider from Google notified us that the exploit will be fixed soon and it will be possible to solve reCAPTCHA only on the corresponding domain.

That's why we ask you to change the algorithm of your captcha upload and start to send URLs in your requests. This can be done with "pageurl" parameter. For detailed description of API parameters please refer to the page: https://2captcha.com/newapi-recaptcha-en

**Comment by acymap** [comment]
No error on capcha solving for me...

[17:15:24] Making account #1
[17:15:24]   Username: ********
[17:15:24]   Password: *********
[17:15:24]   Email   : xxxxxxx+xxxxxxx0020@outlook.fr
[17:16:21]     [DEBUG]:[Worker 0]:  initializing..
[17:16:21]     [DEBUG]:[Worker 0]: Attempting to create user *******0020:***********. Opening browser...
[17:16:21]     [DEBUG]:[Worker 0]: Proxy type: 
[17:16:21]     [DEBUG]:[Worker 0]: 2captcha key
[17:16:21]     [DEBUG]:None
[17:16:21]     [DEBUG]:Step 1: Verifying age using birthday: 1958-11-22
[17:16:21]     [DEBUG]:[Worker 0]: trying to execute workaround script
[17:16:21]     [DEBUG]:[Worker 0]: done executing workaround script, submitting dob
[17:16:21]     [DEBUG]:[Worker 0]: dob submitted
[17:16:21]     [DEBUG]:[Worker 0]: Step 2: Entering account details
[17:16:21]     [DEBUG]:[Worker 0]: https://club.pokemon.com/us/pokemon-trainer-club/parents/sign-up
[17:16:21]     [DEBUG]:[Worker 0]: https://club.pokemon.com/us/pokemon-trainer-club/parents/sign-up
[17:16:21]     [DEBUG]:[Worker 0]: Starting autosolve recaptcha
[17:16:21]     [DEBUG]:[Worker 0]: We will wait 10 seconds for captcha to be solved by 2captcha
[17:16:21]     [DEBUG]:[Worker 0]: Captcha still not solved, waiting another 10 seconds.
[17:16:21]     [DEBUG]:[Worker 0]: Captcha still not solved, waiting another 10 seconds.
[17:16:21]     [DEBUG]:[Worker 0]: Captcha still not solved, waiting another 10 seconds.
[17:16:21]     [DEBUG]:[Worker 0]: Solved captcha
[17:16:21]     [DEBUG]:[Worker 0]: trying to submit
[17:16:21]     [DEBUG]:[Worker 0]: submitted
[17:16:21]     [DEBUG]:[Worker 0]: trying to validate response
[17:16:21]     [DEBUG]:[RESPONSE_VALIDATOR]: https://club.pokemon.com/us/pokemon-trainer-club/parents/sign-up
[17:16:21]     [DEBUG]:[RESPONSE_VALIDATOR]: Username already in use
[17:16:21]     [DEBUG]:[Worker 0]: Failed to create user: *******0020exiting...
[17:16:21]     [DEBUG]:[Worker 0]: threw failed to create user exception, terminate

**Comment by ultrafunkamsterdam** [comment]
fucked up.. captcha solved but username already taken 😅

**Comment by Zyrix** [comment]
check out https://github.com/novskey/PalletTown/pull/25
